openff/toolkit/utils/utils.py

```diff
@@ -164,8 +164,33 @@ def get_data_file_path(relative_path):
     return fn
 
 
-def unit_to_string(input_unit):
-    return str(input_unit)
```

Comment:
I found this was never hit in tests, and I'm not quite sure how it would be triggered.